examples/basic/autocomplete.py

```diff
@@ -11,20 +11,9 @@
 
 async def autocomplete_response(
     ctx: crescent.AutocompleteContext, option: hikari.AutocompleteInteractionOption
-) -> Sequence[hikari.CommandChoice]:
+) -> Sequence[Tuple[str, str]]:
     # All the other options are stored in ctx.options
-    return [hikari.CommandChoice(name="Some Option", value="1234")]
-
-
-async def fetch_autocomplete_options(
-    ctx: crescent.AutocompleteContext, option: hikari.AutocompleteInteractionOption
-) -> Sequence[hikari.CommandChoice]:
-    # Return options with snowflakes converted into the option types.
-    # This function can be extremely expensive.
-    _options = await ctx.fetch_options()
-
-    # Return no options.
-    return []
+    return [("Some Option", "1234")]
```

Comment:
Can you add a comment saying which is the key and value?
Also I would like you can add `await ctx.fetch_options()` back to the example.